xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
```

Comment:
(I may regret this) Would it be possible to put all these beamtime level things in their own dictionary, inside the header? This way when people ask for information about their beamtime we can just hand them the contents on their beamtime dictionary. The potential for regret stems from searching in nested dicts requires mongo syntax (I think) which the users are most likely less interested in (especially as I am pushing `bt_piLast' as an important filter for all the fuzzy searching capacity).